github/resource_github_repository.go

```diff
@@ -118,6 +118,11 @@ func resourceGithubRepository() *schema.Resource {
 					ValidateFunc: validation.StringMatch(regexp.MustCompile(`^[a-z0-9][a-z0-9-]*$`), "must include only lowercase alphanumeric characters or hyphens and cannot start with a hyphen"),
 				},
 			},
+			"vulnerability_alerts": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
my only concern here is, as stated, this default value is only applicable to `private` repos, and a user may unintentionally disable alerts for a `public` repo by omitting this attribute in a config and making it through the `apply` step..could we consider leaving this w/o a default or is the general user experience better with having this value in place?

Reply:
> could we consider leaving this w/o a default

👍 my vote goes to removing the default.